bin/handlers-merge.py

```diff
@@ -79,6 +80,26 @@ def merge(left, right):
 
     return left if left == right else [left, right]
 
+
+def clean_spaces(string):
+    r"""
+    Return the string with the trailing whitespace (excluding line separators)
+    from each line removed, as well all trailing whitespace (including line
+    separators) removed from the whole string.
+
+    >>> clean_spaces("1 \n2\t\n3  \n")
+    '1\n2\n3'
+
+    Lines are normalized to end with LF.
+
+    >>> clean_spaces("1\n2\r3\r\n4")
+    '1\n2\n3\n4'
+    """
+
+    lines = string.rstrip().splitlines()
+    return "\n".join(line.rstrip() for line in lines)
```

Comment:
This way the last line is left without LF even when the last line had LF before. For symmetry, maybe add LF to end of every line including last one.